channel-manager-operations/availabilityBlock.md

```diff
@@ -130,36 +130,44 @@ The example shows a valid availability block definition.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2021-12-24"`\). |
-| `end` | `string` | required | End date in format `"yyyy-MM-dd"` \(e.g., `"2021-12-31"`\). |
+| `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2021-12-24"`\).  |
+| `end` | `string` | required | End date \(excluded\) in format `"yyyy-MM-dd"` \(e.g., `"2021-12-31"`\). This date can't be booked for stay anymore. This date is like a reservation departure. |
 | `releasedDate` | `string` | required | End date in format `"yyyy-MM-dd"` \(e.g., `"2021-12-24"`\). |
```

Comment:
I would like to update this field as well to follow the same format, but I note the current description just says "End date", what exactly is the release date?